src/components/Table/components/TableHead/index.tsx

```diff
@@ -56,6 +67,7 @@ export default function TableHead<D extends object>({ headerGroups, styles }: Ta
                                         ) : (
                                             <span>{column.render?.('Header')}</span>
                                         )}
+                                        <div>{column.canFilter ? column.render?.('Filter') : null}</div>
```

Comment:
Could you please help me understand how the value of **canFilter** is determined?

Reply:
Can we enable filter on some of the columns?

Reply:
> Could you please help me understand how the value of **canFilter** is determined?

This is controlled for all rows as we set `disableFilters` prop to false. Also we provide custom **Filter** component in **defaultColumn**.

 > Can we enable filter on some of the columns?

We can control this with `columnDefinitions`:
```js
...
   {
        id: 'id',
        width: 200,
        Header: 'Id',
        accessor: 'id',
        disableFilters: true,
    }
```

Reply:
You can use the **Stack** component with xs spacing to add some vertical spacing here.